### ISSUE TITLE
Add non-2.2 WCAG guidance to website

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system-user-needs#92]
## Why prioritise this?
We’ve added guidance for the new WCAG 2.2 criteria a while ago (which we’ve since mostly removed again). But it’s a bit odd that all of the rest of WCAG has been missing from considering for the guidance.

We could go through all the other success criteria and add relevant bits to the website. As that could be a pretty big project, we can go about it one criterion at a time (or a bunch at a time).

## Supporting evidence
[#4681](https://github.com/alphagov/govuk-design-system/issues/4681)
[#4684](https://github.com/alphagov/govuk-design-system/issues/4684#issuecomment-2901194977)

## Applicable GOV.UK Initiative(s)
- 5B. Drive commonality and reusability through government services

## Feedback from the team
> We’ve explored this last cycle in [Explore adding further guidance on a WCAG criterion to the website · Issue #4681 · alphagov/govuk-design-system](https://github.com/alphagov/govuk-design-system/issues/4681)
> 
> The main outcome is summarised on [Identify where on the website and how new WCAG guidance could be published · Issue #4684 · alphagov/govuk-design-system](https://github.com/alphagov/govuk-design-system/issues/4684#issuecomment-2901194977)
> 
> In short, we found it would be worth doing. And we wouldn’t need to do it for all 56 success criteria, just doing it for the top 5 failing ones would cover 49% of fails.
